third_party/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/TMTensor/IR/TMTensorBase.td

```diff
@@ -43,7 +43,6 @@ def TMTensor_Dialect : Dialect {
     to.
   }];
   let hasCanonicalizer = 1;
-  let useFoldAPI = kEmitFoldAdaptorFolder;
```

Comment:
@jpienaar FYI that we'll need a similar fix on torch side. The `useFoldAPI` is dropped by https://github.com/iree-org/iree-llvm-fork/commit/475bbea5be04146bb9cdd3e0f3c7bc328c744d50